embark.el

```diff
@@ -1438,23 +1435,23 @@ queued most recently to the one queued least recently."
       (minibuffer-quit-recursive-edit)
     (abort-recursive-edit)))
 
-(defun embark--run-action-hooks (hooks action &rest args)
-  "Run HOOKS for ACTION with ARGS.
-The HOOKS argument must be an alist.  The keys t and :always are
+(defun embark--run-action-hooks (hooks action target quit)
+  "Run HOOKS for ACTION.
+The HOOKS argument must be alist.  The keys t and :always are
 treated specially.  The :always hooks are executed always and the
 t hooks are the default hooks, if there are no command-specific
+hooks.  The QUIT, ACTION and TARGET arguments are passed to the
 hooks."
-  (mapc (lambda (h) (apply h action args))
+  (mapc (lambda (h) (apply h :action action :quit quit target))
```

Comment:
This line took a second to read: it looked like a `:target` was missing until I remembered that `:target` itself is a plist.